src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -0,0 +1,322 @@
+/*
+ * Profiler public include file
+ */
+#ifndef __KINESIS_VIDEO_WEBRTCCLIENT_STATS_INCLUDE__
+#define __KINESIS_VIDEO_WEBRTCCLIENT_STATS_INCLUDE__
+
+#pragma once
+
+#ifdef  __cplusplus
+extern "C" {
+#endif
+
+////////////////////////////////////////////////////
+// Public headers
+////////////////////////////////////////////////////
+
+#define MAX_CANDIDATE_ID_LENGTH                   8U
+#define MAX_ADDRESS_LENGTH                       16U
+#define MAX_RELAY_PROTOCOL_LENGTH                 8U
+#define MAX_TLS_VERSION_LENGTH                    8U
+#define MAX_DTLS_CIPHER_LENGTH                    64U
+#define MAX_SRTP_CIPHER_LENGTH                    64U
+#define MAX_TLS_GROUP_LENGHTH                     32U
+#define MAX_PROTOCOL_LENGTH                       8U
+
+#define MAX_STRING_LENGTH                         255U
```

Comment:
Definitely will conflict. Select longer and more localized names. For example, here a better naming might be MAX_STAT_NAME_STRING_LENGTH perhaps..

Also, don't need to specify the U after the constant